drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -40,16 +40,26 @@ local favorites_version = ""
 ---@param sonos_conn SonosConnection
 ---@param namespaces string[]
 ---@param command "subscribe"|"unsubscribe"
-local _update_subscriptions_helper = function(sonos_conn, householdId, playerId, groupId, namespaces, command)
+local _update_subscriptions_helper = function(sonos_conn, householdId, playerId, groupId, namespaces,
+                                              command)
   for _, namespace in ipairs(namespaces) do
+    local wss_msg_header = {
+      namespace = namespace,
+      command = command,
+      householdId = householdId,
+      groupId = groupId,
+      playerId = playerId
+    }
+    local maybe_token, err = sonos_conn.driver:get_oauth_token()
+    if err then
+      log.warn(string.format("notice: get_oauth_token -> %s", err))
+    end
+
+    if maybe_token then
+      wss_msg_header.authorization = string.format("Bearer %s", maybe_token.access_token)
```

Comment:
For the WSS connection, the `Bearer` token is part of the JSON payload's "header", not the HTTP headers. And the `authorization` key *must* be lowercase, according to the docs.